wgsl/index.bs

```diff
@@ -1023,13 +1023,13 @@ Each distinct type parameterization for a type rule is called an <dfn noexport>o
 For example, [[#arithmetic-expr|unary negation]] (an expression of the form `-`|e|)
 has eight overloads, because its type rules are parameterized by a type |T| that can be any of:
 * [=i32=]
-* vec2&lt;i32&gt;
-* vec3&lt;i32&gt;
-* vec4&lt;i32&gt;
+* [=vector|vec2&lt;i32&gt;=]
```

Comment:
One link for the vectors is probably enough and will help prevent readers from thinking that these links might be to different parts of the spec.

Reply:
done.